src/rgw/rgw_admin.cc

```diff
@@ -4115,7 +4115,6 @@ int main(int argc, const char **argv)
 	encode_json("zones", zones, formatter);
 	formatter->close_section();
 	formatter->flush(cout);
-	cout << std::endl;
```

Comment:
If so what about the other instances of `cout << std::endl;` in the same `switch` ? I don't think it is intentional, as it always returns `0` after the break. So you can remove them too.

Reply:
@joscollin done!